### ISSUE TITLE
visual glitch LinkGroupBox

### ISSUE THREAD

**Issue by stax76** [Revan654/staxrip#84]
I see a visual glitch on all LinkGroupBox instances in the main dialog. For a reason I don't understand the base class GroupBox Text property receives the text, I could fix it using Shadows instead of Overrides on the Text property in LinkGroupBox but it remains a mystery why it works in the original code, I don't see that any relating code or project setting has changed. Do you see that issue in the main dialog too? Changing the Left position from 7 to 4 fixes it too.

While searching for the problem I noticed the main form designer won't open but gives an error, do you get that error too? The error is:

> The base class 'StaxRip.UI.FormBase' could not be loaded.

![untitled](https://user-images.githubusercontent.com/11410540/53600579-2f90ab00-3baa-11e9-83cd-423fd9d651b3.png)

**Comment by Revan654** [comment]
> hile searching for the problem I noticed the main form designer won't open but

Yes, I do get : The base class 'StaxRip.UI.FormBase' could not be loaded.

When I first Loaded Project for the first time, I was able to load the designer, Once I compile it the error pops up. Not sure what version of VS this project started on, it could be an issue going from 2015 -> 2017. 

I'm not getting issue with the groupbox. I think bumping the UI factor to 1.05 helps as well. It might be any issue with certain monitor resolutions and there DPI count. 

This is my View from default settings:

![untitled](https://user-images.githubusercontent.com/38091898/53604563-d06b7000-3b82-11e9-8e09-77d0ae291006.jpg)

**Comment by stax76** [comment]
I've loaded both your current code and my old code in VS and the problem occurs only with your code. I couldn't find what's the difference, the related code appears to be identical and my resolution and DPI value are also not changed, it's strange but easy to fix.

Changing following from 7 to 4 fixes it.

https://github.com/Revan654/staxrip/blob/master/UI/Controls/Controls.vb#L722

regarding the form designer, I found the problem, it's related to following settings:

![screenshot 6](https://user-images.githubusercontent.com/11410540/53607450-19d9b080-3bbf-11e9-8a4e-cd053ef92bcf.png)

**Comment by Revan654** [comment]
> 
> 
> I've loaded both your current code and my old code in VS and the problem occurs only with your code. I couldn't find what's the difference, the related code appears to be identical and my resolution and DPI value are also not changed, it's strange but easy to fix.
> 
> Changing following from 7 to 4 fixes it.
> 
> https://github.com/Revan654/staxrip/blob/master/UI/Controls/Controls.vb#L722
> 
> regarding the form designer, I found the problem, it's related to following settings:
> 
> ![screenshot 6](https://user-images.githubusercontent.com/11410540/53607450-19d9b080-3bbf-11e9-8a4e-cd053ef92bcf.png)

I did tweak it and forward it to the pipeline, It strange. Since I tested it on PC that is only 1080p and it looks to render it correctly. If I had to guess it might be due to app.config or manifest file since some of the keys and values have been changed at both my end and in dotnet 4.7 by MS. Winform is definitely starting to show it's age and MS focus is purely on WPF with not many new additions to Winform. Shame you can't just update just update UI and keep the code. So much is different between the two.

I know sometimes settings file needs a refresh to correct certain issue in past builds might create.

I'm still amazed at how much code was written for Staxrip, I don't even want to image how much time this all took.

Hopefully you like some of the changes I made to your software.

**Comment by stax76** [comment]
It could be a compiler bug or something by design maybe caused by a certain project setting (like 'option strict' for instance). I'm not sure I remember the exact difference between Override and Shadows exactly. The Text property is overriden and in the old project the code is executed in the derived class only but not in the base class, in the new project it runs both in the base class and in the derived class which I think is wrong and potentially causes more bugs. When I have time I'll examine it further.

I worked a bit more than 15 years on StaxRip and gave it up mostly because I didn't use it very often, I use it sometimes and not necessary for encoding but for instance to re-mux.

In the beginning it just called native codec dialogs of divx and xvid and the first implementation to support a command line encoder had like 6-8 different code locations for every codec switch which was very tiresome to work with, the newest design has reduced this to only one location making it much easier to fully build and maintain many different encoders, it's one of the better features, other older areas are more ugly.

I'm really glad you maintain it and hope you have fun with it, I noticed you worked on features that are popular and important so yes I like your work.
